.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx --no-install commitlint --edit
```

Comment:
Can we make it consistent on where husky is configured? Your change make commitlint running as `sh`, but the `sh` for pre-commit is now empty and moved to package.json. What's the rationale?

Reply:
I have used [this](https://adarshaacharya.com.np/blog/setup-husky-precommit-hooks) article for implementation with minor modifications.